pkgs/development/python-modules/markdownify/default.nix

```diff
@@ -17,9 +17,11 @@ buildPythonPackage rec {
 
   disabled = pythonOlder "3.7";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-qyV/nmvUB1EYgoooydAvikv+t0IfVYg0qnmy3+syoJg=";
+  src = fetchFromGitHub {
+    owner = "matthewwithanm";
+    repo = "python-markdownify";
+    rev = "refs/tags/${version}";
```

Comment:
Doesn't this work?

```suggestion
    rev = version;
```

Reply:
If there is a branch with the same name, it won't. Better to be precise from the start.

Reply:
Isn't the checksum there to handle such case?  IMHO this is just an obfuscation hurting readability, nothing prevents a branch from being named refs/tags/0.13.1 either.

Reply:
Ha, fair enough.  Thanks for the information.